packages/core/docs/guide/internationalization.md

```diff
@@ -0,0 +1,49 @@
+# Internationalization
+
+By default we are using `nuxt-i18n` module for handling internationalization, but it's not mandatory to use it even if you are using Nuxt.
+
+In order to provide unified way of configuring i18n across the application for different modules and integrations, we have introduced field `i18n` in each modules configuration that has same format as `nuxt-i18n` options. Clearly, it's possible to add there any configuration if there is a necessity and it will be propagated to all other Vue Storefront modules.
```

Comment:
```suggestion
In order to provide a unified way of configuring i18n across the application for different modules and integrations, we have introduced the field `i18n` in each module's configuration that has the same format as `nuxt-i18n` options. Clearly, it's possible to add there any configuration if there is a necessity and it will be propagated to all other Vue Storefront modules.
```